.github/ISSUE_TEMPLATE/bugs-performance-issues.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report (including performance and build issues)
+about: Help us improve the GPflow code base
+labels: bug
```

Comment:
do these automatically add the corresponding github label to the issue?

Reply:
yes!